package/MDAnalysis/topology/PDBParser.py

```diff
@@ -302,15 +304,21 @@ def _parseatoms(self):
                 occupancies.append(float_or_default(line[54:60], 0.0))
                 tempfactors.append(float_or_default(line[60:66], 1.0))  # AKA bfactor
 
-                segids.append(line[66:76].strip())
+                segids.append(line[72:76].strip())
 
         # Warn about wrapped serials
         if self._wrapped_serials:
             warnings.warn("Serial numbers went over 100,000.  "
                           "Higher serials have been guessed")
 
+        # If segids is not equal to chainids, warn the user
+        if any([a != b for a, b in zip(segids, chainids)]):
+            warnings.warn("Segment IDs and Chain IDs are not completely equal.")
+
         # If segids not present, try to use chainids
         if not any(segids):
+            warnings.warn("Segment IDs (columns 73-76) are missing. "
+                          "Try to load from the chain IDs.")
```

Comment:
I don't feel that we need a warning here if we document the fallback. However, I get that it makes sense to tell users when we make decisions. Still, I would not make it a warning and instead use the logger at INFO (or _maybe_ WARN). 

Also, if we keep it, then change the language to

> Setting segids from chainIDs because no segids found in PDB file.

The "Try to load ..." is ambiguous and implies that the user has to try something. However, we are just informing that we _are_ already doing something.